src/http/ngx_http_variables.c

```diff
@@ -1381,6 +1392,42 @@ ngx_http_variable_tcpinfo(ngx_http_request_t *r, ngx_http_variable_value_t *v,
 
 #endif
 
+#if (T_NGX_REQUEST_START_TIME)
+static char  *months[] = { "Jan", "Feb", "Mar", "Apr", "May", "Jun",
+                           "Jul", "Aug", "Sep", "Oct", "Nov", "Dec" };
+
+static ngx_int_t
+ngx_http_variable_request_start_time(ngx_http_request_t *r,
+    ngx_http_variable_value_t *v, uintptr_t data)
+{
+    u_char          *p;
+    ngx_time_t      *tp;
+    ngx_tm_t tm;
```

Comment:
Can the number of spaces be unified here?

Reply:
There was indeed an issue with the alignment, I fixed it